v20.1/known-limitations.md

```diff
@@ -8,6 +8,14 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### Primary key changes and zone configs
+
+If a table has a [zone configuration](configure-zone.html#create-a-replication-zone-for-a-table) and you change the table's primary key with [`ALTER PRIMARY KEY`](alter-primary-key.html), the zone configuration will no longer apply to the table.
```

Comment:
I think this should be "if your indexes have a zone configuration ..."

Reply:
Reworded. PTAL.

Reply:
LGTM thanks!